### PR TITLE
Border Radii

### PR DESCRIPTION
**Pull request by epetrow**
targets: https://github.com/telerik/kendo-themes-private/issues/177

Design Issue: https://github.com/telerik/web-components-ux/issues/943

### CODE REVIEW

**Review by JoomFX — APPROVED** [review]
LGTM

Only [this](https://github.com/telerik/kendo-themes/blob/border-radii/packages/fluent/scss/dataviz/_layout.scss#L395) should probably be changed to `k-border-radius(md)`

File: packages/bootstrap/scss/checkbox/_layout.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import "@progress/kendo-theme-default/scss/checkbox/_layout.scss";
+@import "../core/border-radii/index.import.scss";
```
Comment (inikolova) [inline packages/bootstrap/scss/checkbox/_layout.scss:2]: Do we need this import here?

File: packages/bootstrap/scss/checkbox/_layout.scss [hunk 2]
```diff
@@ -7,13 +8,13 @@
 
     // Override inherited bootstrap border radius
     .k-checkbox.k-rounded-sm {
-        border-radius: k-math-div( $kendo-border-radius-sm, 2 );
+        border-radius: calc( var(--kendo-border-radius-sm, 0) / 2 );
```
Comment (inikolova) [inline packages/bootstrap/scss/checkbox/_layout.scss:11]: Could we use calc( #{k-border-radius(sm)} / 2 )?

**Review by zhpenkov — APPROVED** [review]

File: packages/bootstrap/scss/fab/_variables.scss [hunk 3]
```diff
@@ -5,7 +5,7 @@
 $kendo-fab-border-width: 1px !default;
 /// The border radius of the FAB.
 /// @group floating-action-button
-$kendo-fab-border-radius: k-math-div( $kendo-border-radius, 2 ) !default;
+$kendo-fab-border-radius: calc( var(--kendo-border-radius, 0 ) / 2 ) !default;
```
Comment (inikolova) [inline packages/bootstrap/scss/fab/_variables.scss:8]: Could we use calc( #{k-border-radius(md)} / 2 ) here? (for consistency)

File: packages/bootstrap/scss/checkbox/_layout.scss (hunk 1 above)
Comment (epetrow) [reply packages/bootstrap/scss/checkbox/_layout.scss:2]: No, we definitely do not

**Review by inikolova — APPROVED** [review]

**Pull request merged**